### PR TITLE
updating descriptions of safa and symkat, and symkat dependency on safa

### PR DESCRIPTION
**Pull request by damien-pous**

### CODE REVIEW

**Comment by damien-pous** [comment]
I removed the "bug-reports" field, and updated the build/install/uninstall instructions

File: packages/symkat/symkat.1.3/opam [hunk 1]
```diff
@@ -4,9 +4,9 @@ author: "Damien Pous <Damien.Pous@ens-lyon.fr>"
...
 version: "1.3"
 homepage: "http://perso.ens-lyon.fr/damien.pous/symkat/"
-bug-reports: ""
 license: "GNU LGPL v3"
-build: [make "PREFIX=%{prefix}%" "install"]
-remove: [make "PREFIX=%{prefix}%" "uninstall"]
-depends: ["ocamlfind" {build} "safa" {>= "1.3"}]
+build: [make "PREFIX=%{prefix}%"]
+install: [make "PREFIX=%{prefix}%" "install"]
+uninstall: [make "PREFIX=%{prefix}%" "uninstall"]
```
Comment (yallop) [inline packages/symkat/symkat.1.3/opam]: "`uninstall`" should be "`remove`"

**Comment by damien-pous** [comment]
sorry, I misunderstood the warning...

**Comment by yallop** [comment]
I've merged this manually, since there was a conflict.

**Comment by damien-pous** [comment]
thanks for the manual merge!
possibly my fork of opam-repository is out of date (I click on the fork button yesterday I thought it would be fine, but this possibly reused the fork I did one and a half year ago, without updating it...).
would you now how to keep it in sync?
best,
damien

**Comment by damien-pous** [comment]
thanks!

**Pull request merged**